Script/AtomicEditor/editor/Editor.ts

```diff
@@ -26,6 +27,9 @@ class Editor extends Atomic.ScriptObject {
 
         this.editorLicense = new EditorLicense();
 
+        var prefs = new Preferences();
+        prefs.read();
```

Comment:
I am wondering if we should have prefs use a static constructor to instantiate the singleton instead of new'ing it here is an example of a static constructor, which could new the instance:

https://github.com/AtomicGameEngine/AtomicGameEngine/blob/master/Script/AtomicEditor/ui/inspector/InspectorUtils.ts#L4